draft-ietf-rtcweb-jsep.xml

```diff
@@ -2272,11 +2272,9 @@ as specified in <xref target="RFC5245" format="default"/>.</li>
             answer. Any media formats that were not present in the most
             recent answer <bcp14>MUST</bcp14> be added after all existing formats.</li>
             <li>The RTP header extensions <bcp14>MUST</bcp14> only include those that
-            are present in the most recent answer.</li>
+            are supported by the application on the associated transceiver.</li>
```

Comment:
I think this makes sense, assuming the W3C spec can say that calling the JS method `setHeaderExtensionsToNegotiate()` will "Set which RTP header extensions are _supported by the application on the associated transceiver_. This will affect which RTP header extensions are negotiated, as defined by [JSEP reference to here]."

Reply:
Yes, that is what we would say in the WebRTC-Extensions spec.